db/migrate/20220810001027_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :users, null: false, foreign_key: true
```

Comment:
* Also here too please stick to the ERD diagram, as future milestones would depend on your databse being created properly